executor/write.go

```diff
@@ -133,14 +130,6 @@ func updateRecord(ctx context.Context, sctx sessionctx.Context, h kv.Handle, old
 					pkDts = append(pkDts, newData[idxCol.Offset])
 				}
 				tablecodec.TruncateIndexValues(t.Meta(), pkIdx, pkDts)
-				handleBytes, err := codec.EncodeKey(sctx.GetSessionVars().StmtCtx, nil, pkDts...)
```

Comment:
maybe we can cleanup L132-135 and L129 too

Reply:
done